packages/ui/react-ui-theme/src/config/tokens.ts

```diff
@@ -301,6 +302,49 @@ const semanticColors = {
       light: ['red', 700],
       dark: ['red', 300],
     },
+
+    //
+    // Codemirror
```

Comment:
I imagine that we may want to factor many of these out from "cm specific styles" to "general theme styles" over time, but given they're specifically being used for cm editor right now this makes that quite clear 👍

Reply:
Yeah this is an interesting use-case… maybe in a later iteration `react-ui-editor` or MarkdownPlugin can export these sememes and composer could provide it to the Vite config.